pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -81,7 +84,55 @@ var (
 	DefaultCleanPodPolicy = commonv1.CleanPodPolicyNone
 )
 
-// NewReconciler creates a PyTorchJob Reconciler
+// Define all the prometheus counters for mxjobs
+var (
+	mxJobsCreatedCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "training_operator_mxjobs_created_total",
+			Help: "Counts number of mx jobs created",
```

Comment:
Should we keep resource name in Help. WDYT @deepak-muley @Jeffwan ?
```suggestion
			Help: "Counts number of created MXJobs",
```